### DISCUSSION TITLE
Eigene Userrollen definieren

### DISCUSSION THREAD

**Discussion by moritzvolkmar** [Samedis-care/samedis-care-community#77] [Shipped - Live on production]
Wenn man viele Personen einlädt, ist es sehr aufwendig immer wieder die ganzen einzelnen Rechte anzuklicken. Häufig haben verschiedene User in der Einrichtung die gleichen Funktionen und bereits eine interne Rollenbeschreibung. 
Es wäre daher praktisch, wenn man aus den einzelnen Rechten individuelle Rollen definieren und speichern könnte. Beim Einladen eines neuen User könnte man dann einfach nur aus diesen Rollen auswählen.

**Comment by moritzvolkmar** [comment]
Passt auch gut zu #71

**Comment by YvesR** [comment]
Kommt mit den nächsten Erweiterungen für my.samedis.care
